Info 2021-04-17 12:43:17: ZMQ install verified.
Info 2021-04-17 12:43:17: id is 02807dee-84a5-4488-b483-058c25392bb5
Error 2021-04-17 12:43:18: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): [EntryNotFound (FileSystemError): Unable to read file '/root/Escritorio/all_of_ia/reinforcement_learning/.vscode/settings.json' (Error: Unable to resolve non-existing file '/root/Escritorio/all_of_ia/reinforcement_learning/.vscode/settings.json')
	at _handleError (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:86:154942)
	at processTicksAndRejections (internal/process/task_queues.js:97:5)
	at async m.readFile (/root/.vscode/extensions/ms-toolsai.jupyter-2021.5.745244803/out/client/extension.js:49:796765)
	at async g.fixSettingsFile (/root/.vscode/extensions/ms-toolsai.jupyter-2021.5.745244803/out/client/extension.js:47:295052)
	at async Promise.all (index 1)
	at async g.updateSettings (/root/.vscode/extensions/ms-toolsai.jupyter-2021.5.745244803/out/client/extension.js:47:296772)] {
  code: 'FileNotFound'
}
Info 2021-04-17 12:43:18: Executing script head -c 1024 /dev/urandom to generate random bytes
Info 2021-04-17 12:43:18: Generated random key of length 2048
Info 2021-04-17 12:43:18: Successfully generated keyfile with OS PRNG
